### DISCUSSION TITLE
after set http body to json strings,I want to get json string from HTTPServerResponse& response ,how to write code？

### DISCUSSION THREAD

**Discussion by edidada** [pocoproject/poco#4638] [Q&A]
**Is your feature request related to a problem? Please describe.**
after set http body to json strings,I want to get json string from HTTPServerResponse& response ,how to write code？

**Describe the solution you'd like**

**Describe alternatives you've considered**

```cpp
thread_local std::string tls_resp_json_body = "";//define

    tls_resp_json_body = json_body;//set
    std::string responseStr = tls_resp_json_body;//get
```

**Additional context**

**Comment by matejk** [comment]
In what way is this related to Poco?
